cmd/podman/common/create.go

```diff
@@ -638,6 +638,12 @@ func DefineCreateFlags(cmd *cobra.Command, cf *entities.ContainerCreateOptions,
 		)
 		_ = cmd.RegisterFlagCompletionFunc(shmSizeFlagName, completion.AutocompleteNone)
 
+		systemdShmSizeFlagName := "systemd-shm-size"
```

Comment:
Can we call it `shm-size-systemd` instead?  I would find it more intuitive for tab completion.  There's already a `--shm-size` flag, so `--shm-size-systemd` would pop up as well.

Reply:
That actually makes a lot of sense, thanks for pointing it out!